inst/extdata/TADAPriorityCharConvertRef.csv

```diff
@@ -69,3 +69,12 @@ mm,4/30/2024 14:25,in,3.9400E-02,0
 nm,4/30/2024 14:25,in,3.9400E-08,0
 nmi,4/30/2024 14:25,in,7.2900E+04,0
 yd,4/30/2024 14:25,in,3.6000E+01,0
+fg/kg,8/29/2024 12:00,ug/L,1.00E-09,0
+g/g,8/29/2024 12:00,ug/L,1.00E-03,0
+ug/kg,8/29/2024 12:00,ug/L,1.00E+00,0
+fg/kg,8/29/2024 12:00,mg/L,1.00E-12,0
+g/g,8/29/2024 12:00,mg/L,1.00E-06,0
+ug/kg,8/29/2024 12:00,mg/L,1.00E-03,0
+#/100mL,8/29/2024 12:00,CFU/100ML,1.00E+00,0
+%,8/29/2024 12:00,ug/L,1.00E-03,0
+%,8/29/2024 12:00,mg/L,1.00E-06,0
```

Comment:
In test data sets, I am still seeing a number of mass/mass units that are not converting because conversion factors to common mass/volume units are not included in TADAPriorityCharConvertRef.

For example: ug/g, mg/kg, ng/g, ng/kg

I think looking at the "Concentration Percentage" units in the WQX MeasureUnit.csv can help generate a list of the missing ones. It would be great to include conversion factors for ug/L and mg/L (as you already have for fg/kg, g/g, ug/kg, etc.)